tests/unit/test_stages/test_base_stage.py

```diff
@@ -151,6 +147,23 @@ def test_dashboard_view(self, workgroups, target_students, state, stats):
         self.assertDictionaryEqual(args[1], expected_context)
 
     @ddt.data(
+        (make_stats(0.4, 0.3, 0.4), (40, 30, 40)),
+        (make_stats(0.1, 0.1, 0.8), (80, 10, 10)),
+        (make_stats(None, None, None), (None, None, None))
+    )
+    @ddt.unpack
+    def test_make_human_stats(self, stats, human_stats_data):
+        stats_order = (StageState.NOT_STARTED, StageState.INCOMPLETE, StageState.COMPLETED)
+        expected_human_stats = BaseGroupActivityStage.make_human_stats(stats)
+
+        self.assertEqual(len(expected_human_stats), len(stats_order))
+
+        for idx, human_stat in enumerate(expected_human_stats.items()):
```

Comment:
Strictly speaking it was tested two lines above: first we check that there is expected count of keys, then we check that each of these keys have proper value. However I fixed the logic to be more straightforward and better fit into actual, expected logic.